integrations/lancedb.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: integration
+name: LanceDB Haystack
+description: A DocumentStore backed by LanceDB
+authors:
+    - name: Alan Meeson
+      socials:
+        github: alanmeeson
+pypi: https://pypi.org/project/lancedb-haystack/
+repo: https://github.com/alanmeeson/lancedb-haystack
+type: Document Store
+report_issue: https://github.com/alanmeeson/lancedb-haystack/issues
+version: Haystack 2.0
+toc: true
+---
+### **Table of Contents**
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+- [License](#license)
+
+## Overview
+LanceDB-Haystack is an embedded [LanceDB](https://lancedb.github.io/lancedb/) backed Document Store for [Haystack 2.X](https://github.com/deepset-ai/haystack/).
+
+## Installation
+
+The current simplest way to get LanceDB-Haystack is to install from GitHub via pip:
+
+```pip install lancedb-haystack```
```

Comment:
```suggestion
```shell
pip install lancedb-haystack

```

Reply:
It's quite hard to show it with github's markdown syntax. This is what I mean
![image](https://github.com/deepset-ai/haystack-integrations/assets/25897116/47280383-f192-4349-97af-9b0f7a9570d4)

Reply:
Updated :heavy_check_mark: